queries.sql

```diff
@@ -32,4 +32,89 @@ SELECT * FROM animals
 /* Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
 SELECT * FROM animals
 	WHERE weight_kg >= 10.4
-		AND weight_kg <= 17.3;
\ No newline at end of file
+		AND weight_kg <= 17.3;
+
+/* 	Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+	Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+	Commit the transaction. */
+BEGIN;		
+UPDATE animals 
+	SET species = CASE
+					WHEN name LIKE '%mon'
+						THEN 'digimon'
+					ELSE
+						'pokemon'
+					END;
+SELECT species FROM animals;
+
+COMMIT;
+
+SELECT species from animals;
+
+
+/* Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made.
+	Then roll back the change and verify that the species columns went back to the state before the transaction */
+BEGIN;
+SAVEPOINT trs_upd_spec_unde;
+UPDATE animals SET species = 'unspecified';
```

Comment:
* Good job working on the requested changes, just some few requirements are left
* Here also you are [required](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified.%20Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.) to confirm the following. you can use the same HINT as before for this too 
* Also Kindly differentiate the transactions for clarity the queries should be arranged properly, and ensure only related transaction queries are written together 
 > Verify that the change was made.

> roll back the change 

> verify that the species columns went back to the state before the transaction.

Reply:
Thanks for the information. I will update it